packages/runtime/src/widgets/DataGrid/DataGrid.tsx

```diff
@@ -277,7 +277,7 @@ export const DataGrid: React.FC<GridProps> = (props) => {
       const { scrollTop, scrollHeight, clientHeight } = container;
 
       // Check if the user has scrolled to the bottom
-      if (scrollTop + clientHeight === scrollHeight) {
+      if (Math.ceil(scrollTop + clientHeight) >= scrollHeight) {
```

Comment:
please add this fix for onScrollEnd in GridView as well

Reply:
Added